client/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react'
+import icon from '../../images/icon.svg'
+
+import './Header.scss'
+
+export interface HeaderProps {
+  title?: string
+  subtitle?: string
+}
+
+const Header: FC<HeaderProps> = ({ title, subtitle }) => (
```

Comment:
A good practice imo is to almost always have an optionnal `className` prop that is merged with the static class of the component.

This allows the parent component to add style for layouting purpose. An example on how to do this exists in the `Button` component.

Reply:
Ok!

Is there a way to do something similar to:

```
<header className={[className, 'Header']}>
```

Or is it mandatory to create a classnames variable (like in the `Button` component)?

Reply:
Well we will always need the `classnames` utility but maybe we can do something like by "inlining" the `classnames` call.

```jsx
import cls from 'classnames'

const Button = ({ className }) => (
  <button className={cls('Button', className)} />
)
```

Reply:
Mmh I prefer to have a separate variable for `classnames`.